BitwardenShared/UI/Vault/Vault/VaultList/VaultListProcessorTests.swift

```diff
@@ -399,7 +399,7 @@ class VaultListProcessorTests: BitwardenTestCase { // swiftlint:disable:this typ
     func test_perform_dismissSetUpUnlockActionCard_error() async {
         await subject.perform(.dismissImportLoginsActionCard)
 
-        XCTAssertEqual(coordinator.alertShown, [.defaultAlert(title: Localizations.anErrorHasOccurred)])
+        XCTAssertEqual(coordinator.alertShown, [.defaultAlert(error: StateServiceError.noActiveAccount)])
```

Comment:
🤔 Given that we have a specific error in mind, it might be worth noting the specific error in the test comments.

Reply:
I guess I'm not necessarily looking for this specific error, really any error should create an alert and error log, it just happens that this is the easiest error to generate. But I can note this error in the test comments if it's helpful.

Reply:
Ah, okay, then that's fine.